Goobi/src/de/sub/goobi/persistence/HibernateUtilOld.java

```diff
@@ -37,6 +37,8 @@
 import org.hibernate.auction.exceptions.InfrastructureException;
 import org.hibernate.cfg.Configuration;
 
+import com.sun.org.apache.bcel.internal.generic.NEW;
+
```

Comment:
No. I guess it came in by the Eclipse auto-complete feature when typing **new** Object()….
It’s ugly, sometimes.